modules/rating/src/main/Glicko.scala

```diff
@@ -33,7 +35,10 @@ case class Glicko(
       volatility > 0 &&
       volatility < (Glicko.maxVolatility * 2)
 
-  def cap = copy(volatility = volatility min Glicko.maxVolatility)
+  // for rating refunds, aging should somehow increase the rating deviation (hence abs)
```

Comment:
I'm confused by this comment -- is the age function specifically for refunds?  If not, where does this function come from?